_schemaV2.graphql

```diff
@@ -2463,6 +2463,11 @@ type Artwork implements Node & Searchable & Sellable {
   # Formatted artwork metadata, including artist, title, date and partner; e.g., 'Andy Warhol, Truck, 1980, Westward Gallery'.
   formattedMetadata: String
   framed: ArtworkInfoRow
+    @deprecated(reason: "Consider using isFramed field (boolean) instead")
```

Comment:
Maybe there is no need for this deprecated message though

Reply:
Yeah, I think the usage of this object involves some consolidation of display text rather than simply representing the boolean state. So it doesn't feel like a 1:1 match where a deprecation of one preferring the other is needed.

So I agree, we can probably leave this deprecation notice out.